### ISSUE TITLE
How to upgrade to self-compiled firmware ？

### ISSUE THREAD

**Issue by RottenLeaf24** [dangowrt/owrt-ubi-installer#93]
I upgraded my router using the itb image from the release, but I can no longer use the bin image to upgrade.
I want to compile some software myself and I end up with an image called ```openwrt-mediatek-mt7622-linksys_e8450-squashfs-sysupgrade.bin```, how can I use it for firmware upgrade?

**Comment by dangowrt** [comment]
No, the `openwrt-mediatek-mt7622-linksys_e8450-squashfs-sysupgrade.bin` image is generated for the device when running the stock bootloader.
You have to select the UBI variant in menuconfig, that will then result in a file called `openwrt-mediatek-mt7622-linksys_e8450-ubi-squashfs-sysupgrade.itb` which works with sysupgrade.

**Comment by RottenLeaf24** [comment]
> No, the `openwrt-mediatek-mt7622-linksys_e8450-squashfs-sysupgrade.bin` image is generated for the device when running the stock bootloader. You have to select the UBI variant in menuconfig, that will then result in a file called `openwrt-mediatek-mt7622-linksys_e8450-ubi-squashfs-sysupgrade.itb` which works with sysupgrade.

I also want to build an image of itb，But the target image only has these kinds in the picture below:
<img width="558" alt="image" src="https://user-images.githubusercontent.com/41962159/164220633-5da86b31-d571-47a9-b0dc-d7f3e4d26ff7.png">
I don't know how to build an image in itb format？

**Comment by dangowrt** [comment]
As I said, you just have to select the right `Target Profile`:
![screenshot](https://user-images.githubusercontent.com/9948313/164224762-a82ffa01-1fad-45cd-a808-5d77b663a4e7.png)

**Comment by RottenLeaf24** [comment]
@dangowrt 
Thank you very much for your help,I found It should select "Multiple Devices" in Target Profile first.
<img width="687" alt="image" src="https://user-images.githubusercontent.com/41962159/164228429-d00cdc55-90dd-4ee1-8c74-276a5204fa52.png">
And then I can see the ubi option in the target devices.
<img width="860" alt="image" src="https://user-images.githubusercontent.com/41962159/164228544-c8a5a490-7c6c-481e-b0df-3b0b6d3417aa.png">

I have compiled the firmware in itb format，It's really appreciate for solving my problem.
